Makefile

```diff
@@ -768,6 +768,7 @@ deploy/service: deploy/envoy deploy/route
 		-p MAS_SSO_ENABLE_AUTH="${MAS_SSO_ENABLE_AUTH}" \
 		-p MAS_SSO_BASE_URL="$(MAS_SSO_BASE_URL)" \
 		-p MAS_SSO_REALM="$(MAS_SSO_REALM)" \
+		-p SPECIAL_MANAGEMENT_ORG_ID="${SPECIAL_MANAGEMENT_ORG_ID}" \
```

Comment:
Can we provide a default for just like https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/2c19e350d9b8e70ead439167ce69ad1b86f69886/Makefile#L707 or the rest of the variables below?

Reply:
I am not sure we can specify defaults for this. as this is a part of the oc process command. Also, There are no defaults for other parameters , they take env value or are hard coded. WDYT?

Reply:
Hey, so basically if you do something like what's done for these variables: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/2c19e350d9b8e70ead439167ce69ad1b86f69886/Makefile#L710-L748, it should default to the value you'll give it. If you do not supply the default value, it'll be an empty string and it is a bad dev experience IMO.

> Also, There are no defaults for other parameters , they take env value or are hard coded

Yes, the hardcode values are the default values. See the linked make target above. So, let's hardcode the default `SPECIAL_MANAGEMENT_ORG_ID `value in the make file like:
```
deploy/service: SPECIAL_MANAGEMENT_ORG_ID ?= "13640203"
```

Reply:
ah sorry, understood you comment now, i got confused since it was pointing to line inside oc process -f , will update the same now

Reply:
fixed.